packages/frontend/app/components/program-year/objectives.js

```diff
@@ -21,11 +21,16 @@ export default class ProgramYearObjectivesComponent extends Component {
     return this.objectiveCount > 0;
   }
 
-  @action
-  load(element, [programYear]) {
-    this.objectiveCount = programYear.hasMany('programYearObjectives').ids().length;
+  constructor() {
```

Comment:
this can be further simplified. we don't need to set `objectiveCount` as component attribute.

instead, a getter would suffice here:

```suggestion
  get objectivesCount() {
    return this.args.programYear.hasMany('programYearObjectives').ids().length;
  }
```

then, remove the tracked property by the same name further up in the class, and we're good to go.

Reply:
@michaelchadwick my bad on this - i fat fingered the getter name in the suggestion. sorry about this. see new comment below.